types/castv2/index.d.ts

```diff
@@ -0,0 +1,119 @@
+/// <reference types="node" />
+
+import { EventEmitter } from "events";
+import { TlsOptions } from "tls";
```

Comment:
```suggestion
```
Unused?

Reply:
Yes, thank you for catching that.  An earlier version had types for the Server class of the module, but I dropped those because I had no way to verify them against my own projects.